docs/releases/3.1.txt

```diff
@@ -130,7 +130,7 @@ File Uploads
 Forms
 ~~~~~
 
-* ...
+* :attr:`django.forms.DateTimeField` accepts dates in ISO 8601 format.
```

Comment:
I think this is a major feature. Maybe it should go near the top? 🙂

If not, perhaps expanding a little bit...